[e2e] TestEnterpriseLicenseSingle is failing

License is not reconciled:

```
[2019-11-22T12:54:07.822Z]     --- FAIL: TestEnterpriseLicenseSingle/Elasticsearch_license_should_be_[gold_platinum] (300.00s)
[2019-11-22T12:54:07.822Z]         utils.go:80:
[2019-11-22T12:54:07.822Z]             	Error Trace:	utils.go:80
[2019-11-22T12:54:07.822Z]             	Error:      	Received unexpected error:
[2019-11-22T12:54:07.822Z]             	            	expectedTypes license type [gold platinum] got basic
[2019-11-22T12:54:07.822Z]             	Test:       	TestEnterpriseLicenseSingle/Elasticsearch_license_should_be_[gold_platinum]
```

```
{"level":"info","@timestamp":"2019-11-22T12:48:44.225Z","logger":"license-controller","message":"Starting reconciliation run","ver":"1.0.0-beta1-35465741","iteration":244,"namespace":"e2e-8ctd3-mercury","name":"test-es-license-provisioning-6wps"}
{"level":"error","@timestamp":"2019-11-22T12:48:44.225Z","logger":"license","message":"while checking license validity","ver":"1.0.0-beta1-35465741","error":"asn1: syntax error: sequence truncated","stacktrace":"github.com/go-logr/zapr.(*zapLogger).Error\n\t/go/pkg/mod/github.com/go-logr/zapr@v0.1.0/zapr.go:128\ngithub.com/elastic/cloud-on-k8s/pkg/controller/common/license.filterValid\n\t/go/src/github.com/elastic/cloud-on-k8s/pkg/controller/common/license/match.go:65\ngithub.com/elastic/cloud-on-k8s/pkg/controller/common/license.bestMatchAt\n\t/go/src/github.com/elastic/cloud-on-k8s/pkg/controller/common/license/match.go:42\ngithub.com/elastic/cloud-on-k8s/pkg/controller/common/license.BestMatch\n\t/go/src/github.com/elastic/cloud-on-k8s/pkg/controller/common/license/match.go:28\ngithub.com/elastic/cloud-on-k8s/pkg/controller/license.findLicense\n\t/go/src/github.com/elastic/cloud-on-k8s/pkg/controller/license/license_controller.go:151\ngithub.com/elastic/cloud-on-k8s/pkg/controller/license.(*ReconcileLicenses).reconcileClusterLicense\n\t/go/src/github.com/elastic/cloud-on-k8s/pkg/controller/license/license_controller.go:203\ngithub.com/elastic/cloud-on-k8s/pkg/controller/license.(*ReconcileLicenses).reconcileInternal\n\t/go/src/github.com/elastic/cloud-on-k8s/pkg/controller/license/license_controller.go:237\ngithub.com/elastic/cloud-on-k8s/pkg/controller/license.(*ReconcileLicenses).Reconcile\n\t/go/src/github.com/elastic/cloud-on-k8s/pkg/controller/license/license_controller.go:50\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler\n\t/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.3.0/pkg/internal/controller/controller.go:216\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem\n\t/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.3.0/pkg/internal/controller/controller.go:192\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).worker\n\t/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.3.0/pkg/internal/controller/controller.go:171\nk8s.io/apimachinery/pkg/util/wait.JitterUntil.func1\n\t/go/pkg/mod/k8s.io/apimachinery@v0.0.0-20190817020851-f2f3a405f61d/pkg/util/wait/wait.go:152\nk8s.io/apimachinery/pkg/util/wait.JitterUntil\n\t/go/pkg/mod/k8s.io/apimachinery@v0.0.0-20190817020851-f2f3a405f61d/pkg/util/wait/wait.go:153\nk8s.io/apimachinery/pkg/util/wait.Until\n\t/go/pkg/mod/k8s.io/apimachinery@v0.0.0-20190817020851-f2f3a405f61d/pkg/util/wait/wait.go:88"}
{"level":"info","@timestamp":"2019-11-22T12:48:44.225Z","logger":"license-controller","message":"Ending reconciliation run","ver":"1.0.0-beta1-35465741","iteration":244,"namespace":"e2e-8ctd3-mercury","name":"test-es-license-provisioning-6wps","took":0.000596409}
{"level":"error","@timestamp":"2019-11-22T12:48:44.225Z","logger":"controller-runtime.controller","message":"Reconciler error","ver":"1.0.0-beta1-35465741","controller":"license-controller","request":"e2e-8ctd3-mercury/test-es-license-provisioning-6wps","error":"no matching license found","errorCauses":[{"error":"no matching license found"}],"stacktrace":"github.com/go-logr/zapr.(*zapLogger).Error\n\t/go/pkg/mod/github.com/go-logr/zapr@v0.1.0/zapr.go:128\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler\n\t/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.3.0/pkg/internal/controller/controller.go:218\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem\n\t/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.3.0/pkg/internal/controller/controller.go:192\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).worker\n\t/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.3.0/pkg/internal/controller/controller.go:171\nk8s.io/apimachinery/pkg/util/wait.JitterUntil.func1\n\t/go/pkg/mod/k8s.io/apimachinery@v0.0.0-20190817020851-f2f3a405f61d/pkg/util/wait/wait.go:152\nk8s.io/apimachinery/pkg/util/wait.JitterUntil\n\t/go/pkg/mod/k8s.io/apimachinery@v0.0.0-20190817020851-f2f3a405f61d/pkg/util/wait/wait.go:153\nk8s.io/apimachinery/pkg/util/wait.Until\n\t/go/pkg/mod/k8s.io/apimachinery@v0.0.0-20190817020851-f2f3a405f61d/pkg/util/wait/wait.go:88"}
```

My feeling _(but I have no proof)_ is that the public key is not correctly set in the binary of the operator.

--- a/build/ci/nightly/Jenkinsfile
+++ b/build/ci/nightly/Jenkinsfile
@@ -33,6 +33,7 @@ REGISTRY = docker.elastic.co
 REPOSITORY = eck-snapshots
 IMG_NAME = eck-operator
 SNAPSHOT = true
+GO_TAGS = release
 LICENSE_PUBKEY = /go/src/github.com/elastic/cloud-on-k8s/build/ci/license.key
 IMG_SUFFIX = -ci
 ELASTIC_DOCKER_LOGIN = eckadmin

